release-notes/index.mdx

```diff
@@ -1,6 +1,7 @@
 ---
-title: "Release Notes"
+title: "Rascal MPL - Release Notes"
+description: "Releases and changelogs for Rascal MPL"
 sidebar_position: 0
 ---
 
-On the left you can find all our (recent) release notes.
\ No newline at end of file
+On the left you can find all our (recent) release notes.xx
```

Comment:
I think the xx is not intended to be committed?

Reply:
that's an oops. Good catch.